api/controllers/API/ConditionDocumentsController.js

```diff
@@ -0,0 +1,80 @@
+/**
+ * ConditionDocumentsController
+ *
+ * @description :: Server-side actions for handling incoming requests.
+ * @help        :: See https://sailsjs.com/docs/concepts/actions
+ */
+
+module.exports = {
+  index: async function (req, res) {
```

Comment:
Request: Add a few lines of documentation on how to call this API function (params, body, etc.), what it returns (for example, why does it return the first condition's documents as opposed to those of the second or higher condition), and why someone might call it. This might be less obvious than other controllers that grab/save data and toss it on a page.

Reply:
done